pkgs/tools/security/sudo-rs/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib
+, bash
+, fetchCrate
+, fetchpatch
+, installShellFiles
+, pam
+, pandoc
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sudo-rs";
+  version = "0.2.0";
+
+  src = fetchCrate {
```

Comment:
is there a reason you chose fetchCrate instead of fetchFromGitHub ?
the github fetch should make the upstream code easier to inspect.

Reply:
fetchCrate is definitely harder for anyone to inspect, but if you don't feel like doing it, it can be done sometimes later or never, I don't think it's a big deal.